app/pb_kits/playbook/pb_caption/_caption.scss

```diff
@@ -16,4 +16,7 @@
   &[class*=dark]{
     @include caption_dark
   }
+  &[class*=dark]{
+    @include caption_dark
+  }
```

Comment:
Looks like this sneaked in. Looks like a dup of the above and in need of removal.